src/screens/Settings/SelectFiatUnit.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+import React, { Component } from "react";
+import { View, FlatList } from "react-native";
+import { connect } from "react-redux";
+import { listFiats } from "@ledgerhq/currencies";
+import UnitRow from "../../components/UnitRow";
+import { saveSettings } from "../../actions/settings";
+import type { SaveSettings } from "../../actions/settings";
+
+const mapDispatchToProps = {
+  saveSettings
+};
+
+class SelectFiatUnit extends Component<{
+  navigation: *,
+  saveSettings: SaveSettings
+}> {
+  static navigationOptions = {
+    title: "Countervalue currency"
+  };
+
+  onPress = (item: *) => {
+    const { navigation, saveSettings } = this.props;
+
+    saveSettings({ counterValue: item.code });
+    navigation.goBack();
+  };
+
+  renderItem = ({ item }) => (
+    <UnitRow unit={item} onPress={() => this.onPress(item)} />
```

Comment:
this is a global polish, but it would be better if we could render `<UnitRow unit={item} onPress={this.onPress} />` and that inside UnitRow implementation, we would pass `unit` by convention. that way, we don't recreate a function.

this is a general tip to try to use everywhere, especially when using PureComponent-s, but not really important here specifically.